streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -38,7 +38,6 @@
 import org.apache.kafka.streams.processor.internals.ProcessorNode;
 import org.apache.kafka.streams.processor.internals.SourceNode;
 import org.apache.kafka.streams.state.KeyValueStore;
-import org.apache.kafka.streams.state.ReadOnlyKeyValueStore;
```

Comment:
weird that the import became unused, even though there were no other code changes...